src/components/manage/Sharing/Sharing.jsx

```diff
@@ -412,8 +416,21 @@ class SharingComponent extends Component {
                           )}
                           {typeof entry.roles[role.id] === 'boolean' && (
                             <Checkbox
+                              name={this.props.intl.formatMessage(
+                                messages.assignNewRoles,
+                                {
+                                  entry: entry.title,
+                                  role: role.id,
+                                },
+                              )}
+                              aria-label={this.props.intl.formatMessage(
+                                messages.assignNewRoles,
+                                {
+                                  entry: entry.title,
+                                  role: role.id,
+                                },
+                              )}
                               onChange={this.onChange}
-                              value={`${entry.id}:${role.id}`}
```

Comment:
why you have removed property 'value'? 
i think it's wrong

Reply:
@giuliaghisini By mistake, I didn't want to delete.